.github/workflows/build_and_push_latest_image.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: "Build and push to quay on every commit"
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: "Build the midstream image inside Dockerfiles"
+        env:
+          QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+          QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+        run: |
+          podman --version
+          cd Dockerfiles 
+          podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11
```

Comment:
@samvarankashyap  Could you explain the benefit of this file?
We will use always the old tag of operator playbooks and running it for every commit would not be very useful.

Reply:
I have changed it in recent pull request https://github.com/redhat-operator-ecosystem/operator-test-playbooks/pull/133